litmus-portal/okteto-dev-env-setup/README.md

```diff
@@ -114,13 +108,13 @@ This directory contains setup guide to start developing Litmus Portal on Okteto
 
   > login as default user with username as `admin` and password as `litmus` 
 
-- <h4>STEP-10:</h4> After you are done with the code changes, you may stop the development environment using `okteto down` and go to the `okteto-dev-env-setup` directory of the cloned repository, delete the  `env` folder contents, delete the `if block` which was added for okteto dev env setup from `litmus-portal/frontend/src/config/index.ts` file and then enter the root directory of the cloned repository i.e. `litmus` and push the changes to your forked repository.
+- <h4>STEP-9:</h4> After you are done with the code changes, you may stop the development environment using `okteto down` and go to the `okteto-dev-env-setup` directory of the cloned repository, delete the  `my-env` folder and the `if block` which was added for okteto dev env setup from `litmus-portal/frontend/src/config/index.ts` file and then enter the root directory of the cloned repository i.e. `litmus` and push the changes to your forked repository.
```

Comment:
do we still need this step if we don't have to modify the readme.md?

Reply:
No, it has been removed with the latest commit, we just have 7 steps now. @rberrelleza